.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
I think this file should rather go into the .gitignore, there's a limit to how many config files we can realistically have in root, so things that aren't really needed for the repo should probably go in .gitignore.